etc/solr-config/access/conf/elevate.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
what is this file for?

Reply:
Its a default solr config file that I copied over from one of the other projects, but it looks like the reference to it is commented out, so we don't need it here, I'll remove it